samples/java_springboot/07.using-adaptive-cards/README.md

```diff
@@ -42,18 +44,62 @@ A message exchange between user and bot can contain media attachments, such as c
+
+Record the `appid` from the returned JSON
+
+### 4. Create the Azure resources
+Replace the values for `<appid>`, `<appsecret>`, `<botname>`, and `<groupname>` in the following commands:
+
+#### To a new Resource Group
+```
+az deployment sub create --name "adaptiveCardsBotDeploy" --location "westus" --template-file ".\deploymentTemplates\template-with-new-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" botSku=S1 newAppServicePlanName="adaptiveCardsBotPlan" newWebAppName="adaptiveCardsBot" groupLocation="westus" newAppServicePlanLocation="westus"
+```
+
+#### To an existing Resource Group
+```
+az deployment group create --resource-group "<groupname>" --template-file ".\deploymentTemplates\template-with-preexisting-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" newWebAppName="adaptiveCardsBotBot" newAppServicePlanName="adaptiveCardsBotPlan" appServicePlanLocation="westus" --name "adaptiveCardsBotBot"
+```
+
+### 5. Update app id and password
+In src/main/resources/application.properties update 
+  - `MicrosoftAppPassword` with the botsecret value
+  - `MicrosoftAppId` with the appid from the first step
+
+### 6. Deploy the code
+- Execute `mvn clean package` 
+- Execute `mvn azure-webapp:deploy -Dgroupname="<groupname>" -Dbotname="<bot-app-service-name>"`
+
+If the deployment is successful, you will be able to test it via "Test in Web Chat" from the Azure Portal using the "Bot Channel Registration" for the bot.
+
+After the bot is deployed, you only need to execute #6 if you make changes to the bot.
 
 ## Further reading
 
-- [Bot Framework Documentation](https://docs.botframework.com)
+- [Spring Boot](https://spring.io/projects/spring-boot)
+- [Maven Plugin for Azure App Service](https://github.com/microsoft/azure-maven-plugins/tree/develop/azure-webapp-maven-plugin)
 - [Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
 - [Adaptive Cards](https://adaptivecards.io/)
 - [Send an Adaptive card](https://docs.microsoft.com/en-us/azure/bot-service/nodejs/bot-builder-nodejs-send-rich-cards?view=azure-bot-service-3.0&viewFallbackFrom=azure-bot-service-4.0#send-an-adaptive-card)
 - [Activity processing](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-concept-activity-processing?view=azure-bot-service-4.0)
 - [Azure Bot Service Introduction](https://docs.microsoft.com/azure/bot-service/bot-service-overview-introduction?view=azure-bot-service-4.0)
 - [Azure Bot Service Documentation](https://docs.microsoft.com/azure/bot-service/?view=azure-bot-service-4.0)
-- [.NET Core CLI tools](https://docs.microsoft.com/en-us/dotnet/core/tools/?tabs=netcore2x)
-- [Azure CLI](https://docs.microsoft.com/cli/azure/?view=azure-cli-latest)
 - [Azure Portal](https://portal.azure.com)
 - [Channels and Bot Connector Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-concepts?view=azure-bot-service-4.0)
```

Comment:
We are adding the `Azure for Java cloud developers` link too (see [here](https://github.com/microsoft/BotBuilder-Samples/blob/Java-R13/samples/java_springboot/03.welcome-user/README.md#further-reading))